sn_registers/src/register.rs

```diff
@@ -36,31 +36,144 @@ pub struct Register {
     permissions: Permissions,
 }
 
+/// A Signed Register on the SAFE Network
+/// This cryptographically secure version of the Register is used to make sure that the data cannot be tampered with
+#[derive(Clone, Debug, Serialize, Deserialize, PartialOrd, PartialEq, Eq, Hash)]
+pub struct SignedRegister {
```

Comment:
for the naming structure, it seems we shall have:
SignedRegister, UnSignedRegister then both using BaseRegister as their base (or UnSignedRegister is the same to BaseRegister)

Or, from user perspective, we shall have the naming of:
PrivateRegister, PulicRegister, and both using BaseRegister as their base

Reply:
> UnSignedRegister is the same to BaseRegister

Yes

> Or, from user perspective, we shall have the naming of:
> PrivateRegister, PulicRegister, and both using BaseRegister as their base

We already use the Private and Public nomenclature for permissions, using it also for Signed and Raw register might be a bit dangerous